### ISSUE TITLE
不收敛，loss下降到15左右就震荡不动了

### ISSUE THREAD

**Issue by mengpengfei** [meijieru/crnn.pytorch#237]
自己训练模型不收敛怎么回事啊， loss从90多下降到15左右时，一直在15附近徘徊，识别率为0

**Comment by mengpengfei** [comment]
是训练时间太短导致，开始时很快，后面非常慢，比用caffe收敛速度要慢不少，两个数量级了

**Comment by Zhang-O** [comment]
My training loss was around  2000 at the beginning , then  reduced to 4 gradually after 1 epoch.

**Comment by LW-CVer** [comment]
image = torch.FloatTensor(opt.batchSize, 3, opt.imgH, opt.imgH)
text = torch.IntTensor(opt.batchSize * 5)
第一行，3应该改成1，因为train.py里面初始化模型用的1通道，然后text我觉得应该要大于你的序列长度才可以，我改为opt.batchSize * 10，最后使用的是adadelta，初始学习率调整到0.1，就能迅速收敛，使用默认学习率直接陷入局部最优，跳不出来，loss一直为17左右

**Comment by LW-CVer** [comment]
image = torch.FloatTensor(opt.batchSize, 3, opt.imgH, opt.imgH)
text = torch.IntTensor(opt.batchSize * 5)
第一行，3应该改成1，因为train.py里面初始化模型用的1通道，然后第二个opt.imgH应该改为opt.imgW，然后text我觉得应该要大于你的序列长度才可以，我改为opt.batchSize * 10z，最后使用的是adadelta，初始学习率调整到0.1，就能迅速收敛，如果使用默认学习率直接陷入局部最优，跳不出来，loss一直为17左右

**Comment by zsc2003** [comment]
> image = torch.FloatTensor(opt.batchSize, 3, opt.imgH, opt.imgH) text = torch.IntTensor(opt.batchSize * 5) 第一行，3应该改成1，因为train.py里面初始化模型用的1通道，然后第二个opt.imgH应该改为opt.imgW，然后text我觉得应该要大于你的序列长度才可以，我改为opt.batchSize * 10z，最后使用的是adadelta，初始学习率调整到0.1，就能迅速收敛，如果使用默认学习率直接陷入局部最优，跳不出来，loss一直为17左右

太感谢了！尤其是adadelta初始学习率调整到0.1这个，卡住了好久都训不出来，learning rate一调直接正确率起飞！

**Comment by Zhang-O** [comment]
> > image = torch.FloatTensor(opt.batchSize, 3, opt.imgH, opt.imgH) text = torch.IntTensor(opt.batchSize * 5) 第一行，3应该改成1，因为train.py里面初始化模型用的1通道，然后第二个opt.imgH应该改为opt.imgW，然后text我觉得应该要大于你的序列长度才可以，我改为opt.batchSize * 10z，最后使用的是adadelta，初始学习率调整到0.1，就能迅速收敛，如果使用默认学习率直接陷入局部最优，跳不出来，loss一直为17左右
> 
> 太感谢了！尤其是adadelta初始学习率调整到0.1这个，卡住了好久都训不出来，learning rate一调直接正确率起飞！

现在还在用crnn吗? 真坚挺啊.

**Comment by zsc2003** [comment]
> > > image = torch.FloatTensor(opt.batchSize, 3, opt.imgH, opt.imgH) text = torch.IntTensor(opt.batchSize * 5) 第一行，3应该改成1，因为train.py里面初始化模型用的1通道，然后第二个opt.imgH应该改为opt.imgW，然后text我觉得应该要大于你的序列长度才可以，我改为opt.batchSize * 10z，最后使用的是adadelta，初始学习率调整到0.1，就能迅速收敛，如果使用默认学习率直接陷入局部最优，跳不出来，loss一直为17左右
> > 
> > 
> > 太感谢了！尤其是adadelta初始学习率调整到0.1这个，卡住了好久都训不出来，learning rate一调直接正确率起飞！
> 
> 现在还在用crnn吗? 真坚挺啊.

hhh course project 想尝试复现一下，一直卡住了。感谢您的帮助~
